extensions/2.0/Khronos/KHR_audio_emitter/README.md

```diff
@@ -0,0 +1,371 @@
+| **sources**    | `number[]` | An array of audio source indices used by the audio emitter. This array may be empty.                | []                   |
+| **positional** | `object`   | A sub-JSON containing the positional audio emitter properties.                                      | {}                   |
+
+#### `type`
+
+Specifies the audio emitter type. This property is required.
+
+- `global` Global audio emitters are not affected by the position of audio listeners. All `positional` properties may not be defined on global audio emitters.
+- `positional` Positional audio emitters play audio at a position in the scene. The properties are defined in the `positional` object. Using sound cones, the orientation is `-Z` having the same emission direction as [`KHR_lights_punctual`](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_lights_punctual) and [glTF cameras](https://github.com/KhronosGroup/glTF-Tutorials/blob/master/gltfTutorial/gltfTutorial_016_Cameras.md).
+
+#### `gain`
+
+Unitless linear multiplier against audio source volume used for determining audio emitter loudness. If not specified, the audio emitter volume gain is `1.0`.
+
+This value is linear, a value of `0.0` is no volume, `0.5` is half volume, `1.0` is the original volume, `2.0` is double the volume, etc. The final volume of the audio is a combination of this value, the audio source's gain, and if the audio emitter is positional, the relative positions of the emitter and listener.
+
+#### `sources`
+
+An array of audio source indices used by the audio emitter. This array may be empty. If empty or not specified, this emitter can be used to define how audio should emit from a node, but not which audio source to play.
+
+#### `positional`
+
+A sub-JSON object containing the positional audio emitter properties. This may only be defined if `type` is set to `positional`.
+
+### Positional Audio Emitter Properties
+
+When the audio emitter type is set to `positional`, additional properties may be defined on the `positional` object.
+
+#### Property Summary
+
+|                    | Type     | Description                                                                                                         | Default value                       |
+| ------------------ | -------- | ------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
+| **coneInnerAngle** | `number` | The anglular diameter of a cone inside of which there will be no angular volume reduction.                          | 6.2831853... (τ or 2π rad, 360 deg) |
+| **coneOuterAngle** | `number` | The anglular diameter of a cone outside of which the volume will be reduced to a constant value of `coneOuterGain`. | 6.2831853... (τ or 2π rad, 360 deg) |
```

Comment:
Consider naming these `innerConeAngle` and `outerConeAngle` for consistency with KHR_lights_punctual? See:

https://github.com/KhronosGroup/glTF/blob/main/specification/2.0/ObjectModel.adoc#52-khr_lights_punctual

Reply:
The original spec was drafted to mirror Web Audio API given the wide acceptance of the property names and behaviors. If there is a KHR reason to align these property names I can get behind it, but my personal feelings are that this goes against what a lot of people have mostly agreed on and understand relative to audio properties. 

Here's the PannerNode docs we referenced when drafting the original OMI_audio_emitter spec https://developer.mozilla.org/en-US/docs/Web/API/PannerNode

Reply:
Are these names are used verbatim outside of the web specification? I do feel that clear inconsistencies among glTF specifications should outweigh borrowing terms verbatim from a particular existing specification.

I have no strong preference about `distanceMaximum` or `range` in the comment thread below – these aren't strong matches to existing glTF terms, and there's already some inconsistency in `accessor.max` vs. `.iridescenceThicknessMaximum`. OK with me to keep those as-is if you prefer.

But asking implementors to use both `innerConeAngle` and `coneInnerAngle` when implementing lights and audio  ... I do not love that. 😅

Reply:
I'm not sure that parity with lights is actually going to be an issue for implementers. The code for these should be very separated. I can see it as a cosmetic issue for consistency between specs, but not much more than that.

I find `coneInnerAngle` easier to read and it's consistent with Web Audio. However, if Khronos wants us to switch to `innerConeAngle` for consistency with lights, it's easy to make that change. We just need the people in charge to make the decision one way or the other. We also might want to get input from Mozilla or Google.